src/Compilers/Core/Portable/Symbols/IParameterSymbol.cs

```diff
@@ -91,8 +91,9 @@ public interface IParameterSymbol : ISymbol
         /// source or metadata.
         /// </summary>
         new IParameterSymbol OriginalDefinition { get; }
+
         /// <summary>
-        /// True if the compiler will synthesize a null check for this parameter (the parameter is declared in source with a '!' following the parameter name).
+        /// True if the compiler will synthesize a null check for this parameter (the parameter is declared in source with a '!!' following the parameter name).
```

Comment:
nit: I don't know whether this should be `<c>` or not. This will render differently on docs.microsoft.com.

Reply:
Perhaps it would make sense to have a tracking issue to verify the rendering/appearance of documentation comments, and perform adjustments like the one you've suggested.